### PR TITLE
squash launch day bugs

### PR DESCRIPTION
**Pull request by allenjprice**

### CODE REVIEW

File: src/cljs/member/services/views.cljs [hunk 1]
```diff
@@ -393,6 +409,7 @@
 
 
 (defn active-order-item [{:keys [fields] :as order} requester]
+  (js/console.log "order is totes " order)
```
Comment (jalehman) [inline src/cljs/member/services/views.cljs]: remove log

File: src/cljs/member/services/views.cljs [hunk 2]
```diff
@@ -332,7 +332,28 @@
     [:span.tag.is-hollow (name status)]))
 
 
-(defn above-the-fold [{:keys [id name date price status billed cancel-btn]} is-open requester]
+(defn- cancel-order-button
+  [{:keys [id type]} account-id loading canceling]
+  (let [opts {:on-click #(dispatch [:services.order/cancel-order id account-id])
+              :type     "danger"
+              :icon     "close"
+              :loading  (if (= @canceling id)
```
Comment (jalehman) [inline src/cljs/member/services/views.cljs]: consolidate to one line

File: src/cljs/member/services/views.cljs [hunk 3]
```diff
@@ -332,7 +332,28 @@
...
+                          @loading
+                          false)}
+         ]
```
Comment (jalehman) [inline src/cljs/member/services/views.cljs]: remove trailing bracket

**Comment by allenjprice** [comment]
@jalehman addressed earlier comments and pushed up the commit that nerfs members cancelling fees on the server. plz to update review 🙏

**Pull request merged**